### DISCUSSION TITLE
Waiting List - Copilot

### DISCUSSION THREAD

**Discussion by sharafcs50** [community/community#20238] [Copilot Conversations] [answered]
Hi, 

My account was already accepted to use copilot on March 26th 2022, now whenever I log in, I am told to join the waiting list again, am I missing something?

`[INFO] [auth] [2022-07-09T13:26:00.417Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2022-07-09T13:26:00.421Z] GitHub Copilot could not connect to server. Extension activation failed: "User not authorized"
`

Thanks,
Sharaf.

**Comment by davecheney** [comment, marked as answer]
Please try visiting https://github.com/settings/copilot and completing the required setup steps.
